lib/ansible/plugins/filter/ipaddr.py

```diff
@@ -90,11 +90,10 @@ def _6to4_query(v, vtype, value):
 
 
 def _ip_query(v):
-    if v.size == 1:
+    if v.size in [1, 2]:
         return str(v.ip)
```

Comment:
Text in controller-side code should be the native text type since `str()` returns `bytes` on Python 2 and `unicode` on Python 3. I know this filter uses `str()` extensively, and that should be changed. But we can start small.
```suggestion
        return to_text(v.ip)
```

Add `from ansible.module_utils._text import to_text`.

Reply:
I'd rather do the `to_text()` fix as a separate PR, and then deal with the all the returns of the ipaddr filter in one go. Feels much more consistent that way.